packages/ui_components_lib/lib/components/common/common_card.dart

```diff
@@ -130,7 +130,7 @@ class CommonCard extends StatelessWidget {
             mainAxisSize: width == null ? MainAxisSize.min : MainAxisSize.max,
             children: [
               if (subtitle != null) subtitle,
-              Flexible(child: centerTitle ? Center(child: title) : title),
+              Expanded(child: centerTitle ? Center(child: title) : title),
```

Comment:
Replacing `Flexible` to `Expanded` means, that our card will fill max width, but in design, such cards are content-wide.
May be we will make it optional with `fillWidth` param and `false` by default?